build-system/pr-check/dist-tests.js

```diff
@@ -48,7 +49,7 @@ function main() {
   const buildTargets = determineBuildTargets();
 
   if (!isTravisPullRequestBuild()) {
-    timedExecOrDie('gulp dist --fortesting --noextensions');
+    downloadDistOutput();
     timedExecOrDie('gulp bundle-size --on_push_build');
     runSinglePassTest_();
```

Comment:
Since this is the last thing that's being done here, there's no need to call `gulp clean` at the end of `runSinglePassTest_()`.

Reply:
Removed.